### ISSUE TITLE
kontext训练后的结果出现横纹

### ISSUE THREAD

**Issue by singlinhaha** [lrzjason/T2ITrainer#42]
训练了一个皮克斯的风格化lora，跑了4000步，出的图有明显的横纹。是底模是nf4的问题吗？还是参数或者数据集的问题？数据集是用gpt4o生成的。

数据集对: 30
rank: 32
repeats: 5
BF: 16
optimizer：adamw
lr_scheduler：cosine_with_restarts
学习率：0.0001
分辨率: 512

![Image](https://github.com/user-attachments/assets/005ae945-2dce-4f1c-93db-20187b0fd9b7)

**Comment by lrzjason** [comment]
可能是模型退化了，试一下早期一点的保存节点看看。

**Comment by singlinhaha** [comment]
> 可能是模型退化了，试一下早期一点的保存节点看看。

发现是用nunchaku加载lora就会出现横纹，如果使用comfyui官方fp8模型，lora好像没起到啥效果

**Comment by GuardSkill** [comment]
我也有横纹，nunchaku和官方的都有横纹 @

**Comment by singlinhaha** [comment]
> 我也有横纹，nunchaku和官方的都有横纹 @
应该是nf4底模的问题，我换成ai-toolkit用原模去训练，然后nunchaku加载，没有出现横纹

**Comment by abozahran** [comment]
can we train on fp8 safetensor file ?

**Comment by starinskycc** [comment]
我用LORA也有条纹，有办法解决么？

**Comment by lrzjason** [comment]
Issue fixed. The main issue of vertical lines occurs is text attention masks which I implemented long time ago. Removed the text attention mask fixed the issue.

修复了该问题，该问题的主因是很久之前实现的text attention mask导致的，目前最新代码已经移除text attention mask相关设置。
